examples/resourceServer/client/index.html

```diff
@@ -1,26 +1,51 @@
 <head>
-    <title>resourceServer</title>
+    <title>ResourceServer</title>
 </head>
 
 <body>
-    <h1>Welcome to resourceServer!</h1>
+    <h1>Welcome to ResourceServer!</h1>
 
     {{> authorize}}
 </body>
 
 <template name="authorize">
+    <p>
+      This <code>meteor-oauth2-server</code> package currently provides functionality for both the AuthenticationServer and the ResourceServer.  This functionality may be split into separate packages and/or examples in the future; however, for the time being, AuthenticationServer and ResourceServer are the same application.
+    </p>
     <p>The following steps represent the steps that are necessary to properly negotiate an oauth2 token. While these
-    steps work on this site, this site is for testing and demo purposes only. Do not use this configuration in production.</p>
-    <p>You'll know if everything is working if you can get to Step 6 and the user ids match.</p>
+    steps work on this site, this site is for testing and demo purposes only. Do not use this configuration in production.  You'll know if everything is working if you can get to Step 6 and the user ids match.</p>
     <hr />
 
-    <h2>Step 0 - Add an an authorized client</h2>
+    <h2>Step C1.1 - Generate Client ID & Secret</h2>
```

Comment:
The purpose of the resourceServer example is to present a dev with a reasonably simple workflow on what they need to implement to get the process working well. The server/index.js already adds a client that is compatible with the clientApplication example. This form adds a confusion to an already complex process. Further, the endpoint for the form doesn't actually insert a client.
I do however see the value of added a representation for each step of the documented oauth2 process. In which case, I recommend this is shunted to a "Config" tab where the dev can reference it on demand and not allow it to confuse the "Authentication" workflow we already have.

So, essentially, the index will have two tabs "Authentication" (default) and "Config". "Authentication" will show the existing authenticate workflow and "Config" will provide the workflow for adding a client.

Thoughts?